---
layout: page
title: Links
nav_order: 6
permalink: /links/
---



### Productivity

#### Data format

Frequently used formats:

- [BAM](https://samtools.github.io/hts-specs/SAMv1.pdf)
- [BED](https://genome.ucsc.edu/FAQ/FAQformat.html#format1)
- [BEDGraph](https://genome.ucsc.edu/FAQ/FAQformat.html#format1.8)
- [bigBED](https://genome.ucsc.edu/FAQ/FAQformat.html#format1.5)
- [FASTQ](https://en.wikipedia.org/wiki/FASTQ_format)
- [GTF](https://www.ensembl.org/info/website/upload/gff.html)
- [HAL](https://genome.ucsc.edu/FAQ/FAQformat.html#format20)
- [MAF](https://genome.ucsc.edu/FAQ/FAQformat.html#format5)
- [SAM](https://samtools.github.io/hts-specs/SAMv1.pdf)
- [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf)



Other relavant information:

- http://gmod.org/wiki/Glossary
- https://genome.ucsc.edu/FAQ/FAQformat.html
- https://m.ensembl.org/info/website/upload/gff.html



#### Software manuals ([Nutstore](https://www.jianguoyun.com/p/DSf1oX4Qu_vMCRi_i6IE) copy)

NGS data

- [hts-specs, samtools, bcftools](https://samtools.github.io/)
- [cutadapt](https://cutadapt.readthedocs.io/en/stable/)
- [STAR](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf)
- [featureCounts](http://subread.sourceforge.net/SubreadUsersGuide.pdf)
- [hisat2](http://daehwankimlab.github.io/hisat2/manual/)
- [stringtie](http://ccb.jhu.edu/software/stringtie/)
- [bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml)
- [pysam](https://pysam.readthedocs.io/en/latest/index.html)



#### Genome Browser

- [UCSC Genome Browser](https://genome.ucsc.edu/)



#### High quality genome annotations

- [Ensembl Genomes](https://ensemblgenomes.org/)
- [Eukaryotic and genomes annotated at NCBI](Eukaryotic genomes annotated at NCBI)



#### Website

- [Explain Shell](https://explainshell.com/)
- [Decoding SAM flags](https://broadinstitute.github.io/picard/explain-flags.html)

<br/>

### Books and Journals

#### Journals in Ecology and Evolution

- [Trends in Ecology and Evolution](https://www.sciencedirect.com/journal/trends-in-ecology-and-evolution)
- [Annual Review of Ecology, Evolution, and Systematics](https://www.annualreviews.org/journal/ecolsys)
- [Nature Review Genetics](https://www.nature.com/nrg/)
- [Nature Genetics](https://www.nature.com/ng/)
- [Molecular Biology and Evolution](https://academic.oup.com/mbe)
- [Nature Ecology & Evolution](https://www.nature.com/natecolevol/)
- [Ecology Letters](https://onlinelibrary.wiley.com/journal/14610248)
- [Molecular Ecology](https://onlinelibrary.wiley.com/journal/1365294x)
- [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/journal/2041210X)
- [Evolution](https://onlinelibrary.wiley.com/journal/15585646)
- [Genetics](https://academic.oup.com/genetics)
- [Genome Biology](https://genomebiology.biomedcentral.com/), [Genome Research](https://genome.cshlp.org/), ...
- [Molecular Cell](https://www.sciencedirect.com/journal/molecular-cell), [Cell Research](https://www.nature.com/cr/), [elife](https://elifesciences.org/), ...
- [Developmental Cell](https://www.sciencedirect.com/journal/developmental-cell), ...
- [Nautre plants](https://www.nature.com/nplants/), [Nature Microbiology](https://www.nature.com/nmicrobiol/), [Molecular Plant](https://www.sciencedirect.com/journal/molecular-plant/), ...
- Other multidisciplinary journals including [Cell](https://www.sciencedirect.com/journal/cell), [Nature](https://www.nature.com/), [Science](https://www.science.org/journal/science), ...



#### Probability and Statistics

- [Introduction to Linear Algebra](http://math.mit.edu/~gs/linearalgebra/)
- [Linear algebra done right](https://link.springer.com/book/10.1007/978-3-319-11080-6)
- [The Elements of Statistical Learning](https://web.stanford.edu/~hastie/ElemStatLearn/)
- [An Introduction to Statistical Learning](https://www.statlearning.com/)
- [Bayesian Data Analysis](http://www.stat.columbia.edu/~gelman/book/)
- [Deep Learning](https://www.deeplearningbook.org/)
- [Modern Statistics for Modern Biology](https://web.stanford.edu/class/bios221/book/index.html)



#### Evolution and Ecology

- [Molecular Evolution: A Statistical Approach](http://abacus.gene.ucl.ac.uk/MESA/) by Ziheng Yang
- [Mathematical Population Genetics](https://link.springer.com/book/10.1007/978-0-387-21822-9) by Warren J. Ewens
- [An Introduction to Population Genetics: Theory and Applications](https://global.oup.com/ushe/product/an-introduction-to-population-genetics-9781605351537) by Rasmus Nielsen & Montgomery Slatkin
- [Theoretical Evolutionary Genetics](https://felsenst.github.io/pgbook/pgbook.html) by J. Felsenstein
- [Probability Models for DNA Sequence Evolution](https://www.springer.com/gp/book/9780387781686) by Richard Durrett
- Coalescent Theory: An Introduction By John Wakeley



#### Other lists of books on Statistics, Evolution, and Bioinformatics

- [https://sgsa.berkeley.edu/current_students/books/](https://sgsa.berkeley.edu/current_students/books/)

<br/>

### Programming

#### R

Style guide: [tidyverse](https://style.tidyverse.org/)

Learning resources:

- [R for data science](https://r4ds.had.co.nz/)
- [Advanced R](https://adv-r.hadley.nz/index.html)
- [data.table](https://github.com/Rdatatable/data.table/wiki/Getting-started)
- [Biostrings](https://bioconductor.org/packages/release/bioc/html/Biostrings.html) and [GenomicFeatures](https://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html)



#### python

Style guide: [PEP](https://www.python.org/dev/peps/pep-0008/), [Google](https://google.github.io/styleguide/pyguide.html)

Learning resources:

- [Python Data Science Handbook](https://github.com/jakevdp/PythonDataScienceHandbook)
- [pandas](https://pandas.pydata.org/)
- [Real Python Tutorials](https://realpython.com/)



#### Linux

There are many reference books on linux:

- UNIX and Linux System Administration Handbook
- Linux Bible
- Linux Administration: A Beginner's Guide
- How Linux Works: What Every Superuser Should Know



-----

Â© 2021 GxE<sup>2</sup> Lab
